assets/src/styles/main.scss

```diff
@@ -36,3 +36,19 @@ body {
         @import './pages/site-location';
     }
 }
+
+#ccsa-area {
+    float: right;
+    margin-top: 43px;
+    margin-right: 10px;
+    font-family: calibri,tahoma,arial,verdana,sans-serif;
+    font-size: xx-large;
+    text-decoration: none;
+
+    a {
+        color: #233f8f !important;
+    }
+    .ngwmn-highlight {
+        color: #d1544a;
+    }
+}
```

Comment:
I was going to suggest you look into using USWDS styles for this, but see you're mimicking the look of the current site, so it makes sense as-is. But for reference, there are SASS mixins and functions that may be used for fonts, sizing, and colors.

Reply:
Rather than floating this left, could you look into using a USWDS grid? With it floating, it behaves unexpectedly when the width is shrunk.
![image](https://user-images.githubusercontent.com/136512/49255675-5cd68d00-f3f2-11e8-8000-dfad15722d0c.png)

![image](https://user-images.githubusercontent.com/136512/49255710-7081f380-f3f2-11e8-8460-5b639d0d6c70.png)

Reply:
good point. I will do that.